pkg/graph/resolvers/trb_request.go

```diff
@@ -54,20 +54,28 @@ func TRBRequestUpdate(ctx context.Context, id uuid.UUID, changes map[string]inte
 
 }
 
-// TRBRequestGetByID returns a TRB request by it's ID
-func TRBRequestGetByID(ctx context.Context, id uuid.UUID, store *storage.Store) (*models.TRBRequest, error) {
+// GetTRBRequestByID returns a TRB request by it's ID
+func GetTRBRequestByID(ctx context.Context, id uuid.UUID, store *storage.Store) (*models.TRBRequest, error) {
 
-	trb, err := store.TRBRequestGetByID(appcontext.ZLogger(ctx), id)
+	trb, err := store.GetTRBRequestByID(appcontext.ZLogger(ctx), id)
 	if err != nil {
 		return nil, err
 	}
+
+	// This could be more efficient by using a join with one single query, and writing logic to
+	// parse out the rows and populate the slice of attendees, but this is a lot simpler
+	attendees, err := store.GetTRBRequestAttendeesByTRBRequestID(ctx, id)
```

Comment:
@jstrang-oddball, I started doing something similar on Mint where you always return the sub fields in the parent , we ended up not taking that approach, and instead leaned into GQL to only return the attendees ( I made a comment below with links to how we do it in MINT).